peps/pep-0778.rst

```diff
@@ -0,0 +1,260 @@
+PEP: 778
+Title: Supporting Symlinks in Wheels
+Author: Ethan Smith <ethan@ethanhs.me>
+Sponsor: Barry Warsaw <barry@python.org>
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
```

Comment:
This is no longer needed and no longer in the template: https://github.com/python/peps/blob/main/peps/pep-0012/pep-NNNN.rst?plain=1
```suggestion
```

Reply:
I must have had an old checkout, thanks for correcting!